back-end/helpers/intent_analyzer.js

```diff
@@ -0,0 +1,38 @@
+export default class IntentAnalyzer{
+  analyzeIntents(models) {
+      const newIntents = [];
+
+      for (const model of models) {
+        newIntents.push(this.formatModel(model))
+      }
+
+      return newIntents;
+    }
+
+  formatModel(model) {//Formats the mongoose model into desired array for IntentLister to use
+    let percentageMap;
+    if (model.total_children == 0) {
+      percentageMap = {"No intents found.": 0};
+    }
+    else {
+      console.log(model);
```

Comment:
Is there a reason we need this line? Otherwise we should remove `console.log()`s.
```suggestion
    
```

Reply:
oops forgot to remove, nice catch thanks